src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -357,6 +388,18 @@
   }
 }
 
+@media (prefers-reduced-motion: no-preference) {
+  @keyframes menu-toggle-m-danger-motion {
+    33% {
+      transform: translate3d(-2px, 0, 0);
```

Comment:
```suggestion
      transform: translateX(-2px);
```

Reply:
On this - won't `translate3d` be hardware accelerated and `translateX` won't? Or is that old/fake news?

Reply:
Since the animation is on a `transform` function, it should already be performant. What I remember about the `translate3d()` (IIRC it was adding `translateZ(0)`, which is benign) deal was adding a transform function to an element would put the element on the GPU layer, so then you could animate other things (other than `translate`) and possibly be more performant? Not sure if it was fake news or not, smells like it, like too good to be true or _"Doctors hate this one CSS trick!"_ 😂

Reply:
Yeah I found a number of sites/articles/blogs saying to use `translate3d` or `translateZ` because it would force it onto the gpu, but none of the usual suspects.